packages/@react-spectrum/dialog/docs/AlertDialog.mdx

```diff
@@ -121,38 +134,68 @@ function Example() {
 ### Variant
 [View guidelines](https://spectrum.adobe.com/page/dialog/#Options)
 
+Confirmation
 ```tsx example
-<DialogTrigger marginEnd="20px">
+<DialogTrigger marginEnd="size-300">
   <ActionButton>Exit</ActionButton>
-  <AlertDialog variant="confirmation" title="Exit instance?" primaryActionLabel="Confirm" cancelLabel="Cancel">
+  <AlertDialog
+    variant="confirmation"
+    title="Exit instance?"
+    primaryActionLabel="Confirm"
+    cancelLabel="Cancel">
     Exit dungeon instance and return to main hub?
   </AlertDialog>
 </DialogTrigger>
-
-<DialogTrigger marginEnd="20px">
+```
+Information
+```tsx example
+<DialogTrigger marginEnd="size-300">
```

Comment:
now that each of these is in its own example, we probably don't need the marginEnd on any of them